### PR TITLE
created the person page on Sanje

### PR DESCRIPTION
**Pull request by SanjeAntona**
added relevant information + applicable parameters

### CODE REVIEW

File: content/about/people/SanjeFenkart/index.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    wp1: true
+    institute: CERN
+    media:
+        email: sanje.antona.fenkart@cern.ch
+        github: SanjeAntona
+        linkedin: sanje-fenkart-656823255?l
+---
+{{< figure src="feature.jpg" alt= "photo of Sanje, (C) Christin Hristova, Christop Markus Rist" class="float-right m-2 w-[30%]" >}} 
+
+Sanje works as a Communications officer for the EVERSE project. She came to CERN in January 2023, where she was first part of the editorial team of the [CERN Courier](https://cerncourier.com), a bi-monthly magazin reporting on particle physics worlwide. Currently, she promotes EVERSE through newsletters, social media and webpage updates.
```
Comment (graeme-a-stewart) [inline content/about/people/SanjeFenkart/index.md:18]: - magazine
- worldwide

File: content/about/people/SanjeFenkart/index.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+Before coming to CERN, Sanje graduated in astrophysics at the [University of Vienna](https://astro.univie.ac.at/en/). During her studies she worked as a science communicator at the planetaria of Vienna and Klagenfurt, and joined a volunteer project  on journalism - [Wolfgang Magazin](https://www.wolfgang-magazin.com/author/sanje-fenkart/) - where she joined the science editorial team.
```
Comment (graeme-a-stewart) [inline content/about/people/SanjeFenkart/index.md:20]: - planetariums
- change the second "joined" to "became a member of"

**Review by graeme-a-stewart — CHANGES_REQUESTED** [review]
A few spelling mistakes!

File: content/about/people/SanjeFenkart/index.md (hunk 1 above)
Comment (SanjeAntona) [reply content/about/people/SanjeFenkart/index.md]: done

File: content/about/people/SanjeFenkart/index.md (hunk 2 above)
Comment (SanjeAntona) [reply content/about/people/SanjeFenkart/index.md]: done

**Review by graeme-a-stewart — APPROVED** [review]

**Pull request merged**